packages/keypair/src/KeypairManagerModal.js

```diff
@@ -167,11 +168,28 @@ export default class KeypairManagerModal extends PureComponent {
   }
 
   renderKeypairRow = keypair => {
+    const nameStyle = {
```

Comment:
please use the className instead of biding inline-style
ref: https://getbootstrap.com/docs/5.1/content/typography/#description-list-alignment